.github/workflows/development_infrastructure.yml

```diff
@@ -1324,19 +1325,14 @@ jobs:
             mongo_connection_string=$(az cosmosdb keys list --name tfs-${{ env.environment }}-mongo --type connection-strings --query "connectionStrings[0].connectionString" -o tsv | sed -e "s/&replicaSet=globaldb//g")
             echo "Mongo connection string: $mongo_connection_string"
 
-            EXTERNAL_API_URL=$(az webapp show --name tfs-${{ env.environment }}-external-api --query defaultHostName -o tsv)
-
             portal_file_storage_key=$(az storage account keys list --account-name tfs${{ env.environment }}storage --query [0].value -o tsv)
 
-            echo "External API url: http://$EXTERNAL_API_URL"
-
             az webapp config connection-string set --name tfs-${{ env.environment }}-dtfs-central-api \
               --connection-string-type custom \
               --settings \
                 CORS_ORIGIN=${{ secrets.CORS_ORIGIN }} \
                 MONGO_INITDB_DATABASE="${{ env.database_name }}" \
-                MONGODB_URI="$mongo_connection_string" \
-                EXTERNAL_API_URL="http://$EXTERNAL_API_URL"
```

Comment:
EXTERNAL_API_URL has been removed, is it not required any more?

Reply:
Same for all of the below.

Reply:
Not for `central-api` no.
It had been removed from the `dev` environment variables, so I checked the javascript codebase and it wasn't there either. Therefore, I updated my Bicep & the CLI scripts here.
I've gone through quite carefully and have pruned everything that is clearly not needed, apart from `CORS_ORIGIN` which I'm still unsure of - it isn't in the export from dev.

I've Slacked you the discrepancies I've found.